### DISCUSSION TITLE
Share Dataset(VID,DET,COCO)

### DISCUSSION THREAD

**Discussion by xyl-507** [STVIR/pysot#496] [Q&A] [answered]
因为某些原因，下载这三个数据集比较费力，不知道作者或者其他大佬能不能分享一下crop好了的数据集(VID,DET,COCO)的百度云链接）

**Comment by xyl-507** [comment]
已经找到百度云链接了。

**Reply by jkboyJohn** [reply]
我也想要，好心人，可以分享一个吗

**Reply by 174614361** [reply]
有 youtube bb 的百度云链接吗？谢谢

**Reply by xyl-507** [reply, marked as answer]
链接：https://pan.baidu.com/s/1WygETKP-9TW8La7EGuD8Mw?pwd=1234 
提取码：1234 
--来自百度网盘超级会员V6的分享
